public/app/AppWrapper.tsx

```diff
@@ -59,6 +59,13 @@ export class AppWrapper extends Component<AppWrapperProps, AppWrapperState> {
     this.setState({ ready: true });
     $('.preloader').remove();
 
+    if (config.featureToggles.multiTenantFrontend) {
+      const fsLoaderEl = document.querySelector('.fs-loader');
+      if (fsLoaderEl) {
+        fsLoaderEl.parentNode?.removeChild(fsLoaderEl);
+      }
+    }
```

Comment:
wdyt about giving both preloaders (single tenant and multi tenant) the same id (`preloader`), then just having one standardised call to remove it? e.g. line 60 could just be `$('#preloader').remove();`